### PR TITLE
Show warning for possibly failing transactions

### PR DESCRIPTION
**Pull request by vyorkin**

### CODE REVIEW

File: src/modules/ethereum/transactions/types.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export interface TransactionObject {
 
 export type StoredTransactions = Array<TransactionObject>;
 
-interface Warning {
+export interface InterpretWarning {
```
Comment (everdimension) [inline src/modules/ethereum/transactions/types.ts:15]: Where is this used?

File: src/ui/pages/SendTransaction/TransactionWarnings/TransactionMayFailWarning.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import ValidationErrorIcon from 'jsx:src/ui/assets/validation-error.svg';
+import { Spacer } from 'src/ui/ui-kit/Spacer';
+import { UIText } from 'src/ui/ui-kit/UIText';
+
+export function TransactionMayFailWarning() {
+  return (
+    <>
```
Comment (everdimension) [inline src/ui/pages/SendTransaction/TransactionWarnings/TransactionMayFailWarning.tsx:10]: This layout needs to be abstracted. It's used in two separate components which are supposed to look the same

**Review by everdimension — CHANGES_REQUESTED** [review]
Where is this used?

File: src/ui/pages/SendTransaction/TransactionWarnings/TransactionMayFailWarning.tsx (hunk 2 above)
Comment (vyorkin) [reply src/ui/pages/SendTransaction/TransactionWarnings/TransactionMayFailWarning.tsx:10]: Agree, I requested review too soon, this should be a draft PR actually... I'll extract the common display logic into `TransactionWarning` component

File: src/modules/ethereum/transactions/types.ts (hunk 1 above)
Comment (vyorkin) [reply src/modules/ethereum/transactions/types.ts:15]: The idea was to display these warnings in `TransactionWarnings` component (before I realized there is no interpretation warnings yet), I'll revert these changes for now

File: src/ui/pages/SendTransaction/TransactionWarnings/TransactionWarnings.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  addressAction: AddressAction | IncomingAddressAction;
+  chain: Chain;
+  networkFeeConfiguration: NetworkFeeConfiguration;
+}) {
+  return (
+    <>
+      {addressAction.transaction.status === 'failed' ? (
+        <TransactionWarning
+          title="Transaction may fail"
+          message="This transaction can not be broadcasted or it may fail during
```
Comment (everdimension) [inline src/ui/pages/SendTransaction/TransactionWarnings/TransactionWarnings.tsx:28]: Why are we not using "message" that comes from interpret response?

**Review by everdimension — APPROVED** [review]

**Pull request merged**